lib/Model.js

```diff
@@ -122,12 +122,16 @@ export default class Model extends ReplicateObject {
     return prediction;
   }
 
-  async createPrediction(input, { webhook, webhookEventsFilter } = {}) {
+  async createPrediction({ input }, { webhook, webhookEventsFilter } = {}) {
```

Comment:
Why does it take two objects? Couldn't this just be one object that looks something like `{input = {}, webhook, webhookEventsFilter}` ?

Reply:
I personally like the interface where the arguments to the prediction are separated from the optional configuration arguments. And I see webhooks as the latter, not the former.

Reply:
What arguments are there to the prediction other than `input`?

```js
createPrediction({input, whatElseWouldGoHere?}, {webhook})
```

Reply:
Nothing yet, but plausibly something in the future, and given `input` is an object with arbitrary keys, I'm not sure how we'd switch from that being the only required argument to there being a `whatElseWouldGoHere` without a new method or people's stuff breaking in subtle ways after a package upgrade.